{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/app.py

```diff
@@ -9,7 +10,8 @@
     autodiscover=True,
     origin="{{cookiecutter.project_slug}}",
     id="1",
-    broker=settings.{{cookiecutter.kafka_server_environment_variable}}, 
+    broker=settings.{{cookiecutter.kafka_server_environment_variable}},
+    store=os.environ.get('STORE_URI', 'memory://'),
```

Comment:
Shall we get the store value from `settings`? Maybe we could add the logic in `settings.py` and here get the value:

```python
# settings.py
{% if cookiecutter.include_rocksdb.lower() == "y" %}
STORE = "rocksdb://"
{% else %}
STORE = "memory://"
{% endif %}

What do you think?

Reply:
Hi @marcosschroh 
Yes i think it is a great idea. I have made the changes.